src/tabs/configuration.html

```diff
@@ -359,7 +359,7 @@
                                 <div class="helpicon cf_tip" i18n_title="configurationArmingHelp"></div>
                             </div>
                             <div class="spacer_box">
-                              <div class="_smallAngle">
+                              <div class="_smallAngle accelNeeded">
```

Comment:
Is this needed? This tag is inside another accelNeeded so hiding the parent I suppose is enough